utils.py

```diff
@@ -22,7 +22,7 @@
 def help(cmd):
 	if cmd.args: # only reply on "!help"
 		return
-	commands = set(cmd.bot.commands.keys())
+	commands = cmd.bot.commands.keys()
```

Comment:
wait we call `commands.remove()` down below

Reply:
Its still a list, so no problem there?

Reply:
ain't no `list`
```py
>>> a = {}
>>> a.keys().remove()
AttributeError: 'dict_keys' object has no attribute 'remove'
```

Reply:
replaced with `del commands[name]`

Reply:
```py
>>> a = {'cow': 'moo'}
>>> c = a.keys()
>>> del c['cow']
TypeError: 'dict_keys' object does not support item deletion
```

Reply:
converting to a list it is